### PR TITLE
Refactoring and module integration

### PR DESCRIPTION
**Pull request by lkorczowski**
- [x] refactoring `preprocess` to module `data` in covid19lib
- [x] including pytest suite to start coverage
- [x] automatic script from *.zip to preprocessed *.csv
- [ ] ~~CI Github Actions + Covecode~~ (will be in another PR)

### CODE REVIEW

**Review by scrouzet — COMMENTED** [review]
Great. Fine with me to merge this when you're ready.

**Review by freddyb2 — COMMENTED** [review]
The `dummy.zip` has only one value in its two columns : 
- annee_comptabilisation
- nb_deces

To have a test be a little bit more exhaustive, we could add more possible values for in these columns.

**Review by scrouzet — APPROVED** [review]
I already had a look at it this afternoon. I'm kind of useless with CI, tests, etc so I trust you that all this works well ! (hence no comments on the details from my side)

**Comment by scrouzet** [comment]
@lkorczowski Je merge ?

File: tests/test_data.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+def test_preprocess_data(dummyfolders):
+    """Check if extract correctly .zip to .csv and preprocess data"""
+    extract_archives(dummyfolders["data"] + "dummy.zip", outputdir=dummyfolders["tmp"])
+    preprocess_data("dummy", inputdir=dummyfolders["tmp"], outputdir=dummyfolders["preprocessed_data"])
+    files = [dummyfolders["preprocessed_data"] + "dummy.pkl"]
+    assert all([os.path.exists(file) for file in files])
+    df = pd.read_pickle(files[0])
+    expected_df = pd.read_pickle(dummyfolders["data"] + "dummy.pkl")  # computed on previous master with same dummy.csv
```
Comment (freddyb2) [inline tests/test_data.py:56]: 👍

**Review by freddyb2 — APPROVED** [review]

**Pull request merged**